From 36043f2b7994c013a99071cb5deacb3e55f9a5fc Mon Sep 17 00:00:00 2001
From: Manisha Chinthapally <manisha.chinthapally@intel.com>
Date: Tue, 8 Jan 2019 12:45:07 -0800
Subject: [PATCH 1136/1214] ded support for new hypercall

PROFILING_GET_STATUS is added to list of profiling hypercalls
This is supporting change to get profiling status info

Tracked-On: PKT-1717
Signed-off-by: Manisha Chinthapally <manisha.chinthapally@intel.com>
---
 include/linux/vhm/acrn_hv_defs.h | 17 +++++++++--------
 1 file changed, 9 insertions(+), 8 deletions(-)

diff --git a/include/linux/vhm/acrn_hv_defs.h b/include/linux/vhm/acrn_hv_defs.h
index af00025..c6049d1 100644
--- a/include/linux/vhm/acrn_hv_defs.h
+++ b/include/linux/vhm/acrn_hv_defs.h
@@ -247,13 +247,14 @@ struct hc_api_version {
 
 
 enum profiling_cmd_type {
-       PROFILING_MSR_OPS = 0,
-       PROFILING_GET_VMINFO,
-       PROFILING_GET_VERSION,
-       PROFILING_GET_CONTROL_SWITCH,
-       PROFILING_SET_CONTROL_SWITCH,
-       PROFILING_CONFIG_PMI,
-       PROFILING_CONFIG_VMSWITCH,
-       PROFILING_GET_PCPUID,
+	PROFILING_MSR_OPS = 0,
+	PROFILING_GET_VMINFO,
+	PROFILING_GET_VERSION,
+	PROFILING_GET_CONTROL_SWITCH,
+	PROFILING_SET_CONTROL_SWITCH,
+	PROFILING_CONFIG_PMI,
+	PROFILING_CONFIG_VMSWITCH,
+	PROFILING_GET_PCPUID,
+	PROFILING_GET_STATUS,
 };
 #endif /* ACRN_HV_DEFS_H */
-- 
2.7.4

